src/main/java/pwr/asi/java/hermes/DTOs/MailDTO.java

```diff
@@ -0,0 +1,73 @@
+package pwr.asi.java.hermes.DTOs;
+
+import pwr.asi.java.hermes.entities.Mail;
+import pwr.asi.java.hermes.entities.Template;
+
+import java.util.Date;
+import java.util.Objects;
+
+public class MailDTO {
+
+    private Template formattedTemplate;
+    private int number;
+    private Date creationTime;
```

Comment:
Don't use `Date` anymore. Since Java 8 there are: `LocalDateTime`, `OffsetDateTime` and `ZonedDateTime`. Consider changing it to one of mentioned